api/v1alpha1/customruntimeenvironment_types_test.go

```diff
@@ -227,16 +227,16 @@ func TestAggregatePhase(t *testing.T) {
 			},
 			expectedOutput: PhaseRunning,
 		},
-		"importing-missing-secret": {
-			cnbi: CustomNBImage{
-				Spec: CustomNBImageSpec{
+		"importing_missing_secret": {
```

Comment:
I'd keep dashes instead of underscore for consistency.

Reply:
uhno, we chatted about that earlier, idk how this got lost..